src/iter/parallel.rs

```diff
@@ -0,0 +1,135 @@
+use rayon::iter::ParallelIterator;
+use rayon::iter::internal::{UnindexedProducer, UnindexedConsumer, Folder, bridge_unindexed};
+use super::*;
+
+/// An `ParallelIterator` over a [`BitSetLike`] structure.
+///
+/// [`BitSetLike`]: ../../trait.BitSetLike.html
+pub struct BitParIter<T>(T);
+
+#[cfg(target_pointer_width="64")]
+pub const BITS_IN_USIZE: usize = 64;
```

Comment:
this const is only used in one function, so let's just take it via `size_of` in the code - should be simpler